### PR TITLE
Configurable TSS pre-parameters generation timeout

### PR DESCRIPTION
**Pull request by nkuba**
We used to set a timeout for a pre-parameters generation to a fixed value of `120 seconds`. It felt like a way more than enough when running it on machines with 4+ CPU cores. Unfortunately, it may be to low for less powerful machines. Here we added a possibility to set the timeout to an arbitrary value based on the operator's observations and resources the server has.

The value can be configured in a toml config file under `PreParamsGenerationTimeout`. If not set the default `2 minutes` will be assumed.

We also remove `PreParamsGenerationConcurrency` parameters as it's not longer required after the concurrency fix in `tss-lib`.

### CODE REVIEW

File: configs/config.toml.SAMPLE [hunk 1]
```diff
@@ -34,3 +34,8 @@
 # This is an optional parameter, if not provided concurrency level for TSS protocol
 # will be set to `3` which is a minimum to run pre-parameters generation process.
```
Comment (pdyraga) [inline configs/config.toml.SAMPLE:35]: Is it still valid explanation after https://github.com/keep-network/keep-ecdsa/pull/347

File: configs/config.toml.SAMPLE [hunk 2]
```diff
@@ -34,3 +34,8 @@
...
 #  PreParamsGenerationConcurrency = 3
+# Timeout for TSS protocol pre-parameters generation. The value
+# should be provided based on resources available on the machine running the client.
+# This is an optional parameter, if not provided timeout for TSS protocol
+# pre-parameters generation will be set to `2 minutes`.
+#  PreParamsGenerationTimeout = "2m"
```
Comment (pdyraga) [inline configs/config.toml.SAMPLE:41]: Maybe we should use "2m30s" as an example on how to format this parameter?

File: configs/config.toml.SAMPLE (hunk 1 above)
Comment (nkuba) [reply configs/config.toml.SAMPLE:35]: If the value is <3 it will be handled by tss-lib and set to `3` anyways.

File: pkg/node/params_pool.go [hunk 3]
```diff
@@ -29,13 +45,25 @@ func (n *Node) InitializeTSSPreParamsPool() {
 
 func (t *tssPreParamsPool) pumpPool() {
 	for {
+		logger.Info("generate new tss pre parameters")
```
Comment (pdyraga) [inline pkg/node/params_pool.go:48]: `s/generate/generating`

File: configs/config.toml.SAMPLE (hunk 1 above)
Comment (pdyraga) [reply configs/config.toml.SAMPLE:35]: For 3 how many concurrent generations are started? Worth explaining here what 3 means.

Comment (pdyraga) [reply configs/config.toml.SAMPLE]: Or... we can remove it now with the fix in `tss-lib`.

Comment (nkuba) [reply configs/config.toml.SAMPLE]: We no longer need this property at all as the bug on tss-lib side was fixed https://github.com/keep-network/keep-ecdsa/pull/347

**Pull request merged**